lib/zendesk_apps_support/package.rb

```diff
@@ -226,6 +223,16 @@ def iframe_only?
 
     private
 
+
+    def runtime_translations(translations)
+      result = translations.dup
```

Comment:
yes! Otherwise this method mutates the translations by deleting a bunch of them. IMO that is surprising and not always desired behaviour.